toptal/picasso
# chore(happo): move stories with animations to cypress side

[FX-2825]

### Description

As our [visual tests placing RFC](https://github.com/toptal/picasso/blob/master/docs/decisions/05-visual-tests-placing.md) points out that when we have flaky tests for snapshots, we should move them to the cypress side to eliminate flakiness and have more control. Stories for BarChart and CategoriesChart have animations and these animations can't be disabled globally. So, that's why they should be moved to Cypress side and disabled animations with prop.

BarChart component has some cypress tests already. So, I've just added happoSnapshot lines.

### How to test

- there won't be any BarChart or CategoriesChart snapshot on happo/storybook side anymore

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-2825]: https://toptal-core.atlassian.net/browse/FX-2825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
